Config/Filter.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the `liip/LiipImagineBundle` project.
+ *
+ * (c) https://github.com/liip/LiipImagineBundle/graphs/contributors
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Liip\ImagineBundle\Config;
+
+final class Filter implements FilterInterface
```

Comment:
once you implemented all filters, this will become obsolete, right?

Reply:
done